addon/services/redux.js

```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
+import redux from 'redux';
+import reducers from '../reducers/index';
```

Comment:
IIRC importing them there was for backwards compatibility reasons so that users of older version of ember-redux wouldn't have to change up the way they're importing... but if we're bumping major versions for these sets of changes, I think @brettburley 's approach is valid.